src/main.cpp

```diff
@@ -1603,8 +1603,8 @@ bool AcceptToMemoryPool(
         CTxMemPoolEntry entry(tx, nFees, GetTime(), dPriority, chainActive.Height(), mempool.HasNoInputsOf(tx), fSpendsCoinbase, consensusBranchId);
         unsigned int nSize = entry.GetTxSize();
 
-        // Accept a tx if it contains joinsplits and has at least the default fee specified by z_sendmany.
-        if (tx.vJoinSplit.size() > 0 && nFees >= ASYNC_RPC_OPERATION_DEFAULT_MINERS_FEE) {
+        // Accept a tx if it contains joinsplits and has at least the default fee for z_* operations.
+        if (tx.vJoinSplit.size() > 0 && nFees >= DEFAULT_FEE) {
```

Comment:
Should this be a constant like this, or could we plumb it all the way out to cli args?

Reply:
Given that it can be overridden by the RPC caller already, I don't think it's necessary to have a config parameter.